discretisedfield/mesh.py

```diff
@@ -265,7 +261,8 @@ def bc(self, bc):
     @property
     def cell(self):
         """The cell dimensions of the mesh."""
-        return self._cell
+        cell = np.divide(self.region.edges, self.n).astype(float)
+        return dfu.array2tuple(cell)
```

Comment:
I would return a numpy array and not a tuple.

Reply:
What about `n`?

Reply:
Not sure

Reply:
A numpy array would probably be best.